EDDI/MainWindow.xaml

```diff
@@ -317,8 +317,7 @@
                         <TextBlock Grid.Column="0" Grid.Row="8" Grid.ColumnSpan="2" Margin="5" TextWrapping="Wrap" Text="{x:Static resx:MainWindow.tab_tts_phonetic_speech_desc}" />
                         <Label x:Name="disableSsmltLabel" VerticalAlignment="Top" Grid.Column="0" Grid.Row="9" Margin="0, 5" Content="{x:Static resx:MainWindow.tab_tts_disable_phonetic_speech_label}" />
                         <DockPanel Grid.Column="1" Grid.Row="9" Margin="0, 5">
-                            <CheckBox Grid.Column="1" x:Name="disableSsmlCheckbox" Margin="5" VerticalAlignment="Top" Checked="disableSsmlUpdated" Unchecked="disableSsmlUpdated"/>
-                            <TextBlock Margin="5" HorizontalAlignment="Stretch" TextWrapping="Wrap" Text="{x:Static resx:MainWindow.tab_tts_disable_phonetic_speech_note}" />
```

Comment:
Unintentional delete?
If however it's intentional, can we also remove the RESX entry?

Reply:
Intentional delete. I'll clean up the resx.